recipes/piccolo-theme/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "piccolo-theme" %}
+{% set version = "0.2.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/piccolo_theme-{{ version }}.tar.gz
+  sha256: e1e247abfdcfdd83bd09b9e75d44707ef39cbf0a5f5e0e8cd15c4b2169748b9f
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - rm -f pyproject.toml
```

Comment:
Just curious, why are you removing pyproject.toml? Is it because you don't want to use `wheel`?

Reply:
See:

https://github.com/conda-forge/conda-forge.github.io/issues/1174